test/macros.jl

```diff
@@ -40,6 +40,28 @@
 
 end
 
+@testset "@gapproperty" begin
+
+    """
+        isoddint_GAP(x)
+    
+    Return `true` if the input integer is odd.
+    """
+    @gapproperty isoddint_GAP(x::Int) = GAP.Globals.IsOddInt(x)
```

Comment:
Perhaps there could be a test using `@inferred` to verify it really is type stable and returning a `Bool`?